server/ai_mediaserver.go

```diff
@@ -871,8 +871,9 @@ func (ls *LivepeerServer) UpdateLiveVideo() http.Handler {
 // @Router /live/video-to-video/{stream}/status [get]
 func (ls *LivepeerServer) GetLiveVideoToVideoStatus() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if r.Method != http.MethodOptions {
-			corsHeaders(w, r.Method)
+
+		corsHeaders(w, r.Method)
+		if r.Method == http.MethodOptions {
```

Comment:
Soft suggestion but we could probably tighten up the routing instead to make it more consistent with how it's done elsewhere, eg

```
ls.HTTPMux.Handle("OPTIONS /live/video-to-video/{streamId}/status", ls.WithCode(http.StatusNoContent))
ls.HTTPMux.Handle("GET /live/video-to-video/{streamId}/status", ls.GetLiveVideoToVideoStatus())
```

I believe we can also even omit the GET as long as the OPTION is specified first, and it'll continue to catch the rest of the methods, in case we're actually posting to this endpoint somewhere.

cc @mjh1